owa/app/js/components/manage_patient_identifier_sources/view_patient_identifier_sources/localPoolIdentifiers.jsx

```diff
@@ -146,15 +148,15 @@ export default class ViewLocalPoolIdentifiers extends Component {
                   style={{
```

Comment:
This exact styling exists in the `localIdentifierGenerator.jsx` file.  Would you consider creating a global variable with this settings and then calling/ passing it as props, where else it is needed? 
Also, have you considered using an external style sheet, instead of styling inline?

Reply:
fixed